clusterloader2/testing/density/constraints_100.yaml

```diff
@@ -0,0 +1,21 @@
+heapster:
+  cpuConstraint: 0.12
+  nemoryConstraint: 387973120 #370 * (1024 * 1024)
+kube-proxy:
+  cpuConstraint: 0.01
+  memoryConstraint: 26214400 #25 * (1024 * 1024)
+l7-lb-controller:
+  cpuConstraint: 0.17
+  memoryConstraint: 69206016 #66 * (1024 * 1024)
+coredns:
+  cpuConstraint: 0.13
+  memoryConstraint: 14680064 #14 * (1024 * 1024)
+kube-apiserver:
+  cpuConstraint: 2.4
```

Comment:
Where are those numbers coming from?
These are different than the current ones:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/density.go#L176

Reply:
I think that explains why some numbers are smaller (i.e. the values need to work for larger clusters).
But why some values are higher (e.g. apiserver cpu)?

BTW - if we want to adjust them later, please file an issue for that.

Reply:
> But why some values are higher (e.g. apiserver cpu)?

I investigated this topic some time ago, so I also took original tests into account - during that time clusterloader density test was not stable.

Since two days ago density test is stable. Maybe better option would be just wait till Monday and if everything goes well with the tests, it will better to readjust the value based only on clusterloader results?

Reply:
sgtm

Reply:
I updated the values. All of them are set based on clusterloader density test only.
What I find weird is "kube-controller-manager" cpu usage being much higher than in original test.